resources/services/config/conformance_pack_rule_compliance.go

```diff
@@ -0,0 +1,163 @@
+package config
+
+import (
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/service/configservice"
+	"github.com/aws/aws-sdk-go-v2/service/configservice/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+
+	"github.com/aws/smithy-go"
+)
+
+func ConformancePackRuleCompliance() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_config_conformance_pack_rule_compliances",
```

Comment:
Why not add this as a subresource to `config.conformance_packs`? this way you'd get rid of the 4-level resolver

Reply:
done